go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/google/licenseclassifier/v2 v2.0.0
 	github.com/google/uuid v1.4.0
 	github.com/google/wire v0.5.0
-	github.com/hashicorp/go-getter v1.7.2
+	github.com/hashicorp/go-getter v1.7.4
```

Comment:
Is there a change in response of go-getter from 1.7.2 ->1.7.4?
What was the root cause of the issue mentioned in the card?

Reply:
@theakshaymenon Looks like the wrong ticket is tagged in this one